### PR TITLE
Added rsvp data into response from `POST /event/staff/checkin/`

### PR DESCRIPTION
**Pull request by Nydauron**
I never realized that this was a thing until I realized that `POST /checkin/` did this, and since mobile needs registration data, this allows them to access it without having to request via `GET /registration/USERID/` (which would require to decode the JWT user token)

### CODE REVIEW

**Review by Timothy-Gonzalez — DISMISSED** [review]
LGTM!

Do we have a proper type for rsvp data, or is just variable and that's why we use a generic map?

**Comment by Nydauron** [comment]
> Do we have a proper type for rsvp data, or is just variable and that's why we use a generic map?

It's defined within the config itself similar to the registration data. RSVP data typically will like have the user id, whether the user is attending or not, and the associated registration data of that user.

**Comment by Timothy-Gonzalez** [comment]
> > Do we have a proper type for rsvp data, or is just variable and that's why we use a generic map?
> 
> It's defined within the config itself similar to the registration data. RSVP data typically will like have the user id, whether the user is attending or not, and the associated registration data of that user.

I wonder if there's a clean way to build a type from the config... probably not. That's the only nitpick I had, so should be fine.

**Review by asehgal220 — APPROVED** [review]
Looks great!

File: documentation/docs/reference/services/Event.md [hunk 1]
```diff
@@ -483,6 +488,11 @@ Valid values for `status` are `Success`, `InvalidEventId`, `BadUserToken`, `Alre
     "newPoints": 10,
     "totalPoints": 10,
     "status": "Success"
```
Comment (Nydauron) [inline documentation/docs/reference/services/Event.md:490]:
```suggestion
    "status": "Success",
```

**Pull request merged**